### PR TITLE
Placed google repository as first priority.

### LINKED ISSUE #1336: Place `google()` repository as the first priority repository
## Description
In build. gradle file jcenter() repository have first priority than google() repository. My idea is to Place google() repository as the first priority repository to avoid such errors like:
**Failed to resolve: multidex-instrumentation**
Here is a[ link](https://stackoverflow.com/questions/45608362/android-studio-3-0-beta-1-failed-to-resolve-com-android-supportmultidex1-0-2) to check to solve this error.

### Update [Required]
- [ ] Update the project level build.gradle file and place google() above jcenter().

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

### PR DESCRIPTION
**Pull request by moksh-mahajan**
### Description
Updated build.gradle file to make google repository as first priority.

Fixes #1336 

### Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Application was built successfully in android studio without any errors

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by moksh-mahajan** [comment]
@Rimjhim28 Please review.

File: PowerUp/build.gradle [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 buildscript {
     repositories {
-        jcenter()
-        maven { url 'https://maven.google.com' }
         google()
+        maven { url 'https://maven.google.com' }
+        jcenter()
+
     }
```
Comment (Rimjhim28) [inline PowerUp/build.gradle:8]: Please remove this extra line.

Comment (moksh-mahajan) [reply PowerUp/build.gradle:8]: Done. Please check.

**Review by Rimjhim28 — CHANGES_REQUESTED** [review]
@moksh-mahajan Squash the commits.

**Comment by moksh-mahajan** [comment]
@Rimjhim28 Done. Please review.

**Review by Rimjhim28 — APPROVED** [review]
LGTM

**Pull request merged**